src/views/ChannelView.js

```diff
@@ -19,6 +19,7 @@ const MessageUserProfilePanel = lazy(() =>
 function ChannelView () {
   const { networkStore } = useContext(RootContext)
   const { channel } = useParams()
+  window.lastChannel = (channel == undefined) ? window.lastChannel : channel
```

Comment:
Instead of using `window` I'd recommend using `useHistory` from `react-router-dom` OR using a store from context, just to keep the application state in order.